docs/guides/estimate-job-run-time.mdx

```diff
@@ -59,12 +59,45 @@ Output:
+
+* Session usage is the time from when the first job starts until the session goes inactive, is closed, or when its last job completes, whichever happens last. It includes both classical and quantum time.
+* Batch usage is the sum of quantum time of all jobs in the batch.
+* Single job usage is the quantum time the job uses in processing.
+
+The usage reported on the dashboard or by using the API is the time a QPU is locked for your workload. Failed or canceled jobs count toward your usage in certain circumstances - see the [Failed and canceled jobs](#failed-job) section for details.
+
+Your usage has different impacts, depending on which channel you're using:
+
+- For Qiskit Runtime on IBM Cloud users, the usage determines how much the job costs.  See [Manage cost](/guides/manage-cost#manage-cost) for details.
+- For IBM Quantum Platform users, this translates to shares. The [fair-share scheduler](/guides/fair-share-scheduler) prioritizes instances with the most shares left. Thus, the higher your usage, the longer your next job stays in the queue.
+
+<span id="failed-job"></span>
+### Usage for failed and canceled jobs
+
+When a job is failed or canceled, the reported usage is as follows:
+
+* Job or batch mode: The reported usage is the time the QPU was locked for executing your workload until the time it failed or was canceled. Therefore, if the failure or cancellation occurred before the lock, the reported usage is zero. Otherwise, the workload's reported usage is the value that Qiskit Runtime returns as `consumed`. Thus, some failed jobs do not appear in your reported usage and others do.
+
+* Session mode: The reported usage is the wall-clock time from the when the first job started executing in the session until the session terminates, regardless of the number of jobs that fail or are canceled.
+
+### Determine a workload's actual usage
+
+After a workload has completed, there are several ways to view its actual usage:
+
+- On the IBM Quantum Platform [Workloads table](https://quantum.ibm.com/workloads) in the Usage column.  From the Home page, click *View all* on the Recent workloads table.  The Usage column shows actual usage for completed workloads.
+- Run [`batch.usage()`](/api/qiskit-ibm-runtime/batch#usage) or [`session.usage()`](/api/qiskit-ibm-runtime/session#usage) in `qiskit-ibm-runtime` 0.30 or later.  If using an older version of `qiskit-ibm-runtime` (>= 0.23 and < 0.30), the usage can be still be found in `session.details()["usage_time"]` and `batch.details()["usage_time"]`.
+- Call the [GET usage](/api/runtime/tags/usage#tags__usage) REST API directly. The reported usage is the `elapsed_time` value returned by the `GET /sessions/{id}` endpoint, for both batch and session workloads.
+
 <span id="view-usage"></span>
-## View the estimated workload (job) usage on IBM Quantum Platform
+### View the estimated workload (job) usage
```

Comment:
Shouldn't this be combined with the first section of this page (Estimate job run time)? Also what do you think about renaming this page to `Workload usage`?  Since people should first understand what usage is before trying to estimate it. 

Also this is outside of the scope of this PR, so it can go into another one if you prefer.

Reply:
> Shouldn't this be combined with the first section of this page (Estimate job run time)?

But then we would be telling people how to estimate usage before explaining what usage is?

Reply:
Maybe rearrange the whole thing so we start by explaining what usage is.